### PR TITLE
Add postmark email adapter alternative

### PR DESCRIPTION
**Pull request by glennreyes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,7 +230,7 @@ var server = ParseServer({
 });
 ```
 
-You can also use other email adapters contributed by the community such as [parse-server-sendgrid-adapter](https://www.npmjs.com/package/parse-server-sendgrid-adapter) or [parse-server-mandrill-adapter](https://github.com/back4app/parse-server-mandrill-adapter).
+You can also use other email adapters contributed by the community such as [parse-server-postmark-adapter](https://github.com/glennreyes/parse-server-postmark-adapter), [parse-server-sendgrid-adapter](https://www.npmjs.com/package/parse-server-sendgrid-adapter) or [parse-server-mandrill-adapter](https://github.com/back4app/parse-server-mandrill-adapter).
```
Comment (flovilmart) [inline README.md]: if it's available on NPM, please indicate the npm link :)

Comment (glennreyes) [inline README.md]: Sure ;) will do it for all email adapters.

**Comment by facebook-github-bot-0** [comment]
@glennreyes updated the pull request.

**Comment by facebook-github-bot-8** [comment]
@glennreyes updated the pull request.

**Comment by glennreyes** [comment]
Not sure about that CI build fail 😓 I just updated the `README.md`

**Comment by flovilmart** [comment]
Awesome, didn't knew you added all of those.
As a list it maybe nicer don't you think?

**Comment by glennreyes** [comment]
The other have been there already, I've just updated the links to the npm packages ;).
Yeah, I've thought about doing a list instead. Will update it in a minute.

**Comment by facebook-github-bot-3** [comment]
@glennreyes updated the pull request.

**Comment by facebook-github-bot-8** [comment]
@glennreyes updated the pull request.

**Comment by flovilmart** [comment]
Awesome! 🙏

**Pull request merged**